d2l-visible-on-ancestor-behavior.html

```diff
@@ -0,0 +1,119 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="d2l-dom.html">
+<link rel="import" href="../d2l-fastdom-import/fastdom.html">
+
+<script>
+	window.D2L = window.D2L || {};
+	window.D2L.PolymerBehaviors = window.D2L.PolymerBehaviors || {};
+
+	/** @polymerBehavior */
+	D2L.PolymerBehaviors.VisibleOnAncestorBehavior = {
+
+		properties: {
+
+			/**
+			 * Whether the element is only visible when hovered or focus is within ancestor with 'd2l-visible-on-ancestor-target'.
+			 */
+			visibleOnAncestor: {
+				type: Boolean,
+				reflectToAttribute: true
+			}
+
+		},
+
+		attached: function() {
+
+			if (!this.visibleOnAncestor) return;
```

Comment:
Not sure if you want an attribute change observer on it instead, in case there are cases where folks want to add (or remove) the attribute later?

Reply:
yeah good point.  i will add this.